uint64 timestamp			# time since system start (microseconds)

int8 instance	            # sensor instance

float32 mag_t	            # magnetic field measurement in Tesla

float32 temp_c	            # temperature measurement in deg C
